api/app/signals/apps/api/tests/test_reaction_request_flow_trigger.py

```diff
@@ -30,8 +28,28 @@ def test_email_unknown_no_REACTIE_GEVRAAGD_state_allowed(self):
         self.signal.reporter.save()
         self.signal.refresh_from_db()
 
-        url = self.detail_endpoint.format(pk=self.signal.id)
         payload = {'status': {'state': workflow.REACTIE_GEVRAAGD, 'text': 'Our question.'}}
-        response = self.client.patch(url, data=payload, format='json')
+        response = self.client.patch(f'/signals/v1/private/signals/{self.signal.pk}', data=payload, format='json')
 
         self.assertEqual(response.status_code, 400)
+
+    def test_SIG_4511_update_status_more_than_400_characters(self):
+        data = {
+            'status': {
+                'text': 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut '
+                        'labore et dolore magna aliqua. Morbi tristique senectus et netus et malesuada fames. '
+                        'Vestibulum rhoncus est pellentesque elit ullamcorper dignissim cras tincidunt lobortis. Vitae '
+                        'auctor eu augue ut lectus arcu bibendum at varius. Luctus venenatis lectus magna fringilla '
+                        'urna. Adipiscing commodo elit at imperdiet dui accumsan sit. Venenatis urna cursus eget nunc '
+                        'scelerisque viverra mauris in. Aliquam sem et tortor consequat id porta. Massa enim nec dui '
+                        'nunc. Pellentesque adipiscing commodo elit at imperdiet dui accumsan sit.',
+                'state': 'reaction requested'
+            }
+        }
+
+        response = self.client.patch(f'/signals/v1/private/signals/{self.signal.pk}', data, format='json')
+
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.json()['status']['text'][0],
+                         'Zorg dat deze waarde niet meer dan 400 tekens bevat (het zijn er nu '
```

Comment:
Small aside;

We have to address our error messages, they are a mix of English and Dutch like this one because Django's error messages are translated. Also string matching like this seems brittle, but we do a lot of it in the test suite.

Reply:
Agreed, we already have the wish to make the error messages more uniform. So let's take this in consideration in that PR.